docs/source/parquet.mdx

```diff
@@ -1,13 +1,32 @@
 # List Parquet files
 
-Datasets can be published in any format (CSV, JSONL, directories of images, etc.) to the Hub, and they are easily accessed with the 🤗 [Datasets](https://huggingface.co/docs/datasets/) library. For a more performant experience (especially when it comes to large datasets), Datasets Server automatically converts every dataset to the [Parquet](https://parquet.apache.org/) format. The Parquet files are published to the Hub on a specific `refs/convert/parquet` branch (like this `amazon_polarity` [branch](https://huggingface.co/datasets/amazon_polarity/tree/refs%2Fconvert%2Fparquet) for example) that lives in parallel to the `main` branch.
+Datasets can be published in any format (CSV, JSONL, directories of images, etc.) to the Hub, and they are easily accessed with the 🤗 [Datasets](https://huggingface.co/docs/datasets/) library. For a more performant experience (especially when it comes to large datasets), Datasets Server automatically converts every dataset to the [Parquet](https://parquet.apache.org/) format.
+
+## What is Parquet?
+
+Parquet is a columnar storage format optimized for querying and processing large datasets. Parquet is a popular choice for big data processing and analytics and is widely used for data processing and machine learning.
+
+In Parquet, data is divided in chunks called "row groups", and within each row group, it is stored in columns rather than rows. Each row group column is compressed separately using the most adapted compression algorithm, and contains metadata about the data it contains.
+
+This structure allows for efficient reading and querying of the data:
+- only the necessary columns are read from disk (projection pushdown); no need to read the entire file. This reduces the memory requirement for working with Parquet data. 
+- thanks to the statistics of each row group (min, max, number of NULL values), entire row groups can be skipped if they do not contain the data of interest (automatic filtering)
```

Comment:
```suggestion
- statistics can be calculated from each row group (min, max, number of NULL values), so entire row groups can be skipped if they do not contain the data of interest (automatic filtering).
```

Reply:
reformulated in https://github.com/huggingface/datasets-server/pull/2422/commits/66caeb079a6c68392726a4e3cf555e2f9d0fd574